src/_P078_Eastron.ino

```diff
@@ -173,229 +187,777 @@ boolean Plugin_078(uint8_t function, struct EventStruct *event, String& string)
     }
 
     case PLUGIN_WEBFORM_LOAD:
+    {
+      {
+        const __FlashStringHelper *options_model[6] =
+        { F("SDM220 & SDM120CT & SDM120"), F("SDM230"), F("SDM72D"), F("DDM18SD"), F("SDM630"), F("SDM72D V2") };
+        addFormSelector(F("Model Type"), P078_MODEL_LABEL, 6, options_model, nullptr, P078_MODEL);
+        addFormNote(F("Submit after changing the modell to update Output Configuration."));
+      }
       {
-        {
-          const __FlashStringHelper * options_model[4] = { F("SDM120C"), F("SDM220T"), F("SDM230"), F("SDM630") };
-          addFormSelector(F("Model Type"), P078_MODEL_LABEL, 4, options_model, nullptr, P078_MODEL );
```

Comment:
to be honest I did not take care of the older models that were defined in the original release,
I used the modifed upgraded plugin from pull request #3504 and just added the needed values for the new meter modell, unfortunately my programming skills are not good enough to make it backwards compatible.
Too bad, I really wanted to get this integrated into prod release anyhow, however while testing it on an esp32 it worked really well so far.

Reply:
> ...used the modifed upgraded plugin from pull request...

I got that impression already 😉 that PR has similar issues.
It would be quite a poor experience if, after upgrading ESPEasy to a next release, you are forced to completely re-do your carefully crafted configuration for this plugin...

Your effort is highly appreciated though, but some work in this area will be needed. I'll see if I can be of some assistance.

Reply:
I can rename everything to the orig plugin and thenn I can create completel new models that set the model ID to the ones that react the same ...

Reply:
The work already done does look like an improvement, so throwing that away is not helping anybody. Just adding a conversion for the settings, and adding a flag that this is done, seems more appropriate.

Reply:
It looks like some models were no longer part of the settings.
Not sure if they are now "combined" into others.

Another problem (also with the other pending PR on which this is based) is that there is an awful lot of strings involved.
This plugin will be huge in build size.

Reply:
I am not that pro once it comes to coding
I did some ugly conversation logic see: https://github.com/helbgd/ESPEasy/commit/68dce188b2f6d76b223b50954e7a17fa9bab6e2a
I made some comments in there, all models are currently possible to be chosen that were there even before.